projects/plugins/protect/src/class-jetpack-protect.php

```diff
@@ -214,6 +215,7 @@ public function initial_state() {
 			'apiRoot'            => esc_url_raw( rest_url() ),
 			'apiNonce'           => wp_create_nonce( 'wp_rest' ),
 			'registrationNonce'  => wp_create_nonce( 'jetpack-registration-nonce' ),
+			'credentials'        => Credentials::get_credential_array(),
```

Comment:
Adding credentials to the initial state allows us to access them immediately without an async request in the front end.